src/Parts/Channel.php

```diff
@@ -48,7 +48,7 @@ class Channel
     public ?Bitwise $flags;
     public ?int $total_message_sent;
     /**
-     * @var \Ragnarok\Fenrir\Parts\Tag[]
+     * @var Tag[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)